Content.Server/Traits/Assorted/BrokenLegsComponent.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.Traits.Assorted;
+
+[RegisterComponent, Access(typeof(BrokenLegsSystem))]
+public sealed class BrokenLegsComponent : Component
+{
+    [DataField("walkSpeedModifier"), ViewVariables(VVAccess.ReadWrite)]
+    public int WalkSpeedModifier = 0;
+
+    [DataField("sprintSpeedModifier"), ViewVariables(VVAccess.ReadWrite)]
+    public int SprintSpeedModifier = 0;
+
+    [DataField("accelerationSpeedModifier"), ViewVariables(VVAccess.ReadWrite)]
+    public int AccelerationSpeedModifier = 0;
+
+    [ViewVariables(VVAccess.ReadWrite),
+     DataField("carriageId", customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>))]
+    public string CarriageId = "Carriage";
+}
```

Comment:
Components should be single-purpose, this is doing 2 entirely different things (spawning a vehicle + legspeed).

This should honestly just wait for woundmed and use legspeed there rather than adding a different implementation.